src/utility/constants/APIEndpoints.js

```diff
@@ -0,0 +1,6 @@
+export const APIEndpoints = {
+    GET_SERVICES_LIST: {
+        name: "Get Service",
+        endpoint: "https://3zjz5tihg2.execute-api.us-east-1.amazonaws.com/ropsten",
```

Comment:
Should this be coming from env and not be checked in?

Reply:
deleted